__tests__/app.test.js

```diff
@@ -256,4 +256,80 @@ describe('App', () => {
     
     // need to add a test here for if passed an article id that does not exists (eg. post /api/articles/99999999/comments), can't figure out right now so will come back to this later
   });
+  describe('patch /api/articles/:article_id', () => {
+    test('should respond with status code: 200 and the updated article', () => {
+      const requestBody = {
+        inc_votes: 50
+      }
+      return request(app)
+      .patch('/api/articles/1')
+      .send(requestBody)
+      .expect(200)
+      .then((response) => {
+        const body = response.body
+        expect(typeof body.article.author).toBe('string')
+        expect(typeof body.article.title).toBe('string')
+        expect(typeof body.article.article_id).toBe('number')
+        expect(typeof body.article.topic).toBe('string')
+        expect(typeof body.article.created_at).toBe('string')
+        expect(typeof body.article.votes).toBe('number')
+        expect(typeof body.article.article_img_url).toBe('string')
+        expect(typeof body.article.body).toBe('string')
+      });
+    });
+    
+    test('votes property should be changed by the correct amount', () => {
```

Comment:
To make your test suite extra nice you could  amend the description above and remove the word  "updated" as you are not actually asserting it has been updated in that test. As you are doing it in the below test. The other option is to assert the value of votes on line 275 _instead of_ the below test. 

I think the first option is best 👍